svcs/amigo/table.go

```diff
@@ -49,30 +75,13 @@ func (fd *FrontendData) initTeams(teamId string) []byte {
 	msg := Message{
 		Message: "scoreboard",
 		Values:  Scoreboard{
-			Chals:   chals,
+			Chals:   challenges,
 			TeamRow: rows,
 		},
 	}
 	rawMsg, _ := json.Marshal(msg)
 
 	return rawMsg
-	//teams := fd.ts.GetTeams()
-	//rows := make([]TeamRow, len(teams))
-	//for i, t := range teams {
-	//	r := TeamRowFromTeam(t)
-	//	if t.ID() == teamId {
-	//		r.IsUser = true
-	//	}
-	//	rows[i] = r
-	//}
-	//
-	//msg := Message{
-	//	Message: "teams",
-	//	Values:  rows,
-	//}
-	//rawMsg, _ := json.Marshal(msg)
-	//
-	//return rawMsg
 }
 
 func TeamRowFromTeam(t *store.Team, chals []store.FlagConfig) TeamRow {
```

Comment:
I think, the name of the function is a little bit odd. `TeamInfo` could be much nicer.